spec/requests/v1/artists_request_spec.rb

```diff
@@ -3,10 +3,10 @@
 describe 'artists endpoints' do
   describe 'GET /artists' do
     it 'returns JSON for all artists' do
-      artists = create_list(:artist, 1)
+      artists = create(:artist)
+      songs = create_list(:song, 3, artist: artists)
```

Comment:
Since `create(:artist)` yields a single artist, not a bunch of artists, I'd rename this variable to be singular